src/components/shared/layout/Sidebar.vue

```diff
@@ -144,4 +146,12 @@
             border-radius: 0;
         }
     }
+
+    @media (max-width: 768px) {
+
+        .ui.segments.universal-editor.responsive {
```

Comment:
Should this be on by default if it is a UX concern?
Could possibly have a responsive prop that was true by default so it can be overridden in special cases?

Reply:
What is the offset logic and where is it used?

Reply:
Essentially its the 55px sidebar width which we are offsetting here `$full-screen-breakpoint: $sidebar-max-width + $collapsed-cc-menu-width;`

I have added an additional .no-offset class, in this case it just sets the width to 100vw

Thanks